web/components/templates/dashboard/dashboardPage.tsx

```diff
@@ -115,9 +115,7 @@ const DashboardPage = (props: DashboardPageProps) => {
       : undefined
   );
 
-  const [currFilter, setCurrFilter] = useState(
-    searchParams.get("filter") ?? null
-  );
+  const [currFilter, setCurrFilter] = useState<string | null>("");
```

Comment:
should we use useLocalStorage here? Like if a user is currently using filter `XYZ`, how would we persist this between refreshes?

Reply:
@ScottMktn , we currently persist filter by URL, so if user refresh page we will apply filter from URL, so I don't think we need to store in local storage rn